editor/gui/scene_tree_editor.cpp

```diff
@@ -1602,6 +1627,18 @@ SceneTreeEditor::SceneTreeEditor(bool p_label, bool p_can_rename, bool p_can_ope
 	update_node_tooltip_delay->set_one_shot(true);
 	add_child(update_node_tooltip_delay);
 
+	revoke_dialog = memnew(ConfirmationDialog);
+	revoke_dialog->set_ok_button_text(TTR("Revoke"));
+	add_child(revoke_dialog);
+	revoke_dialog->connect(SceneStringName(confirmed), callable_mp(this, &SceneTreeEditor::_update_ask_before_revoking_unique_name));
+	VBoxContainer *vb = memnew(VBoxContainer);
+	revoke_dialog->add_child(vb);
+	revoke_dialog_label = memnew(Label);
+	vb->add_child(revoke_dialog_label);
+	ask_before_revoke_checkbox = memnew(CheckBox(TTR("Don't Ask Again")));
+	ask_before_revoke_checkbox->set_tooltip_text(TTR("This dialog can also be enabled/disabled in the editor setting: Docks > Scene Tree > Ask Before Revoking Unique Name."));
```

Comment:
My bad, missed this one:
```suggestion
	ask_before_revoke_checkbox->set_tooltip_text(TTR("This dialog can also be enabled/disabled in the editor settings: Docks > Scene Tree > Ask Before Revoking Unique Name."));
```

Reply:
Or "with the editor setting" if preferred.

Reply:
Would this be better?

```
This dialog can also be enabled/disabled in Editor Settings:
```

Reply:
Hm I'm not sure, "the" would still be required IMO.

Reply:
I checked and we have a few "in the Editor Settings" messages, so we should go with that for consistency.
```suggestion
	ask_before_revoke_checkbox->set_tooltip_text(TTR("This dialog can also be enabled/disabled in the Editor Settings: Docks > Scene Tree > Ask Before Revoking Unique Name."));
```

Reply:
There are a couple of "in Editor Settings/Project Settings" too. So it might be good idea to look up what is wanted for consistency.

But for now i will go with your suggestion.